src/genesisStates/kovan.json

```diff
@@ -3,5 +3,5 @@
   "0x0000000000000000000000000000000000000002": "0x1",
   "0x0000000000000000000000000000000000000003": "0x1",
   "0x0000000000000000000000000000000000000004": "0x1",
-  "0x00521965e7bd230323c423d96c657db5b79d099f": "1606938044258990275541962092341162602522202993782792835301376"
+  "0x00521965e7bd230323c423d96c657db5b79d099f": "0x100000000000000000000000000000000000000000000000000"
 }
```

Comment:
Ok.

Reply:
mainnet file comment (no way to attach found due to file size):

Consistency checks, addition/deletion number stays the same, random value tests (full text searches), genesis block hash test execution from Everton.

Looks good.